### DISCUSSION TITLE
how to think about covid test results in risk assessment

### DISCUSSION THREAD

**Discussion by maryalbino** [microCOVID/microCOVID#1165] [Q&A]
We're trying to figure out how many covid tests, which types and at what interval are required for both vaccinated or non vaccinated people to gather in close quarters with the lowest possible risk (as close to zero as we can get)

Which is another way of asking: what's the best way to assess and control for false negatives in covid risk assessment?

**Comment by jackc271** [comment]
Here's my thoughts: Since rapid antigen tests have estimated average 85% sensitivity (source https://www.nytimes.com/article/at-home-covid-tests-accuracy.html and I glanced at some papers that gave estimates across a wide range so this seems reasonable), I think of a negative result as meaning you have roughly 100%-85% = 1/6 the risk at that time. The risk reduction drops off after some time (which is particularly relevant to most PCR tests), I'll try to model that when I have a chance.

**Reply by srhopkins-lab** [reply]
Given that rapid tests are becoming abundant, would be super helpful to incorporate an option for all participants had negative rapid test 1 hour before the activity.
